src/services/BackendService/BackendService.spec.ts

```diff
@@ -133,12 +133,12 @@ describe('BackendService', () => {
 
       expect(covidshield.Upload.create).toHaveBeenCalledWith(
         expect.objectContaining({
-          keys: expect.toHaveLength(14),
+          keys: expect.toHaveLength(28),
         }),
       );
       keys
         .sort((first, second) => second.rollingStartIntervalNumber - first.rollingStartIntervalNumber)
-        .splice(0, 14)
+        .splice(0, 28)
```

Comment:
Would it be worth exporting + importing  MAX_UPLOAD_KEYS here and using the string value ?

```
t('returns last 28 keys if there is more than 28', async () => { 
```